tests/ecosystem/deployment/test_acm.py

```diff
@@ -1,14 +1,15 @@
 from ocs_ci.ocs.acm.acm import import_clusters_with_acm
 from ocs_ci.ocs.acm.acm import AcmAddClusters
-
+from ocs_ci.framework.testlib import acm_import
 
 ####################################################################################################
 # This file is placeholder for calling import ACM as test, until full solution will be implimented #
 ####################################################################################################
 
 
-def test_import_acm(setup_acm_ui):
-    acm_obj = AcmAddClusters(setup_acm_ui)
+@acm_import
+def test_acm_import(setup_acm_ui):
+    AcmAddClusters(setup_acm_ui)
```

Comment:
I dont understand this test file...

Reply:
This is the file for the ACM testing, we will include most likely functions that you want to execute after ACM is installed. Since it is inside the deployment parent directory, the functions here will execute deployment alike code, like the import scenario that is covered in this PR.